### PR TITLE
Homepage content

### PR DESCRIPTION
**Pull request by msecret**
This is the full homepage with the multiple sections, only lacking the footer. Tracks https://github.com/18F/cg-style/pull/10 in cg-style.

### CODE REVIEW

File: pages/home.md [hunk 1]
```diff
@@ -3,5 +3,5 @@ layout: home
 permalink: /
 hero-image: /assets/img/clouds.jpg
 hero-text: "The Government Innovation Platform"
-hero-subtext: "Free your team to focus on creating quality services securely hosted in the cloud"
+hero-subtext: "We built it. We manage it. We keep improving it. Want to use it?"
```
Comment (jcscottiii) [inline pages/home.md]: Was this in the design docs?? maybe I just can't find it lol.

File: js/happyfoxchat.js [hunk 2]
```diff
@@ -1,18 +0,0 @@
-    <!-- HappyFox Live Chat Script -->
-    window.HFCHAT_CONFIG = {
-      EMBED_TOKEN: "995a2db0-46bf-11e5-986c-5b521052aa24",
-      ACCESS_TOKEN: "a5ba09020f0d40ae9eb91e299800cd47",
-      HOST_URL: "https://www.happyfoxchat.com",
-      ASSETS_URL: "https://d1l7z5ofrj6ab8.cloudfront.net/visitor"
-  };
-
```
Comment (jcscottiii) [inline js/happyfoxchat.js:8]: might have to talk to @mogul about removing the chat from the landing page.

File: js/18fanalytics.js [hunk 3]
```diff
@@ -1,10 +0,0 @@
-(function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
-(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
-m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
-})(window,document,'script','https://www.google-analytics.com/analytics.js','ga');
-
```
Comment (jcscottiii) [inline js/18fanalytics.js:5]: need to keep this

**Comment by jcscottiii** [comment]
Looks good @msecret. Had a few suggestions. and additionally some of the final css might change depending on my suggestions here: https://github.com/18F/cg-style/pull/13. Also, lets talk about what you want to do with the jquery removal for the site that's causing the wercker failures.

File: pages/home.md (hunk 1 above)
Comment (dlapiduz) [inline pages/home.md]: This feels a bit pushy... @mogul did you write it?

Comment (msecret) [inline pages/home.md]: It was in James Hupp's document: https://docs.google.com/document/d/1EJEToDGuloXp1GMTqpgdCxLQwF2Z5bopdwGm-UUWMOQ/edit#. It was one of a few options, please pick whichever one you like.

File: js/happyfoxchat.js (hunk 2 above)
Comment (msecret) [inline js/happyfoxchat.js:8]: I can add it back

Comment (mogul) [inline js/happyfoxchat.js:8]: I'm okay with removing it... It's clearly not getting used much. I care more about it on the deck.

File: pages/home.md (hunk 1 above)
Comment (mogul) [inline pages/home.md]: Hey @dlapiduz, why would you think pushy text was me?! _hurt_

File: js/18fanalytics.js (hunk 3 above)
Comment (msecret) [inline js/18fanalytics.js:5]: Is this file actually being imported anywhere? I can't find where it is.

Comment (jcscottiii) [inline js/18fanalytics.js:5]: @geramirez any word on this?

Comment (geramirez) [inline js/18fanalytics.js:5]: as long as we have the same analytics snippet somewhere it should be ok.

Comment (jcscottiii) [inline js/18fanalytics.js:5]: ahh as long as the page loads it. gotcha. thanks @geramirez !! no import needed

Comment (msecret) [inline js/18fanalytics.js:5]: Yea and there is an analytics snippet already existing in javascript.html

**Comment by jcscottiii** [comment]
Looks good @msecret! Thanks for cranking this out so quickly

**Pull request merged**